src/commands/docs/docs.command.ts

```diff
@@ -48,43 +49,70 @@ export class DocsCommand implements CommandClass {
     } else if (args.length > 2) {
       version = ~args.indexOf('gms1') || ~args.indexOf('GMS1') ? 'GMS1' : 'GMS2';
       if (args.indexOf('-i') !== -1) image = true;
+
+      // find a tag
+      whoTag = args.find(arg => {
+        return arg.match(/^<@\d+>$/) !== null;
```

Comment:
Instead of using a regex, you can use [Message.mentions](https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=mentions), which *should* be faster since Discord.js stores that information in each message on receipt anyway.

Reply:
Also see [here](https://discord.js.org/#/docs/main/stable/class/MessageMentions).